### PR TITLE
Implement Panel changes to support internal SFTP subsystem on Daemon

### PR DESCRIPTION
**Pull request by DaneEveritt**
Panel changes required by https://github.com/Pterodactyl/Daemon/pull/51

### CODE REVIEW

File: app/Http/Controllers/API/Remote/SftpController.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            );
+
+            $this->clearLoginAttempts($request);
+        } catch (AuthenticationException $exception) {
+            return response()->json([
+                'error' => 'Invalid credentials.',
+            ], 403);
+        } catch (RecordNotFoundException $exception) {
+            return response()->json([
+                'error' => 'Invalid server.',
```
Comment (lancepioch) [inline app/Http/Controllers/API/Remote/SftpController.php]: Probably should trans these errors.

File: app/Http/Controllers/API/Remote/SftpController.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    /**
+     * Get the throttle key for the given request.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return string
+     */
+    protected function throttleKey(Request $request)
+    {
+        return Str::lower(array_get(explode('.', $request->input('username')), 0) . '|' . $request->ip());
```
Comment (lancepioch) [inline app/Http/Controllers/API/Remote/SftpController.php]: too cool for `strtolower` but you're fine with `array_get` and `explode` 😛

File: resources/themes/pterodactyl/server/settings/sftp.blade.php [hunk 3]
```diff
@@ -66,20 +36,12 @@
                 <div class="form-group">
                     <label for="password" class="control-label">@lang('strings.username')</label>
                     <div>
-                        <input type="text" class="form-control" readonly value="{{ $server->username }}" />
+                        <input type="text" class="form-control" readonly value="{{ auth()->user()->username }}.{{ $server->uuidShort }}" />
```
Comment (lancepioch) [inline resources/themes/pterodactyl/server/settings/sftp.blade.php]: 👍 Now you're getting into the Laravel spirit!

File: app/Http/Controllers/API/Remote/SftpController.php (hunk 2 above)
Comment (DaneEveritt) [reply app/Http/Controllers/API/Remote/SftpController.php]: Don't tell anyone, I just copy/pasted this out of the trait to modify the username attr.

File: app/Http/Controllers/API/Remote/SftpController.php (hunk 1 above)
Comment (DaneEveritt) [reply app/Http/Controllers/API/Remote/SftpController.php]: Good call. Users will never see this anyways (at least currently).

**Pull request merged**